fedora-messaging.spec

```diff
@@ -0,0 +1,132 @@
+%{!?__python2: %global __python2 %__python}
+%{!?python2_sitelib: %global python2_sitelib %(%{__python2} -c "from distutils.sysconfig import get_python_lib; print(get_python_lib())")}
```

Comment:
Does EPEL7 not define these? I think both these macros should be defined everywhere we care about building this.

Reply:
Yeah. And this currently does not build on EPEL7 because of missing dependencies (OpenSSL, blinker, and Twisted for Python 3).
Why do we need PyOpenSSL exactly?

Reply:
I believe Twisted still relies on it for... Something. Possibly certificate chain validation? AFAIK cryptography still hasn't got that API.